.travis.yml

```diff
@@ -124,6 +125,7 @@ before_install:
     - source ~/venv/bin/activate
 
 install:
+    - echo "backend:agg" > $MATPLOTLIBRC/matplotlibrc
```

Comment:
It might clutter up the log a bit less to do this without an envvar:
```sh
mkdir -p ~/.config/matplotlib
echo "backend: agg" > ~/.config/matplotlib/matplotlibrc
```
Looks like there are several ways to set the config: http://matplotlib.org/users/customizing.html#the-matplotlibrc-file